From 23084aaa76d9fc3121ff2ab38c04ee9893793e00 Mon Sep 17 00:00:00 2001
From: NeilBrown <neilb@suse.de>
Date: Mon, 20 Aug 2012 10:33:50 +1000
Subject: mdmon: fix arg processing for -a
Bug-Debian: http://bugs.debian.org/691671
Comment: from stable/bugfix upstream 3.2.6 version
Comment: one of the 3 patches fixing mentioned issues

'-a' was not being recognised as an abbreviation for '--all'.

Signed-off-by: NeilBrown <neilb@suse.de>
---
 mdmon.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mdmon.c b/mdmon.c
index dee02a9..2d0503d 100644
--- a/mdmon.c
+++ b/mdmon.c
@@ -299,7 +299,7 @@ int main(int argc, char *argv[])
 		{NULL, 0, NULL, 0}
 	};
 
-	while ((opt = getopt_long(argc, argv, "th", options, NULL)) != -1) {
+	while ((opt = getopt_long(argc, argv, "tha", options, NULL)) != -1) {
 		switch (opt) {
 		case 'a':
 			container_name = argv[optind-1];
-- 
1.7.10.4

